trajopt_sco/src/osqp_interface.cpp

```diff
@@ -23,6 +23,9 @@ Model::Ptr createOSQPModel()
 }
 
 OSQPModel::OSQPModel()
+  : osqp_workspace_(new OSQPWorkspace, [](OSQPWorkspace* ptr) { osqp_cleanup(ptr); })
```

Comment:
I am not sure what is the best way to avoid using `new`.  Does the following work?
```
osqp_workspace_(std::make_unique<OSQPWorkspace, [](OSQPWorkspace* ptr) { osqp_cleanup(ptr); }>())
```

Reply:
I 'm not sure that I tried. This comment said that it wouldn't.
https://stackoverflow.com/questions/21788066/using-stdmake-unique-with-a-custom-deleter

Reply:
Yep, It looks like it is not possible.